src/js/bookmarklet.js

```diff
@@ -48,8 +48,10 @@ function iFrame(apiRoot) {
     }
 
     function grabCitation(){
-      var cite;
-      document.domain.split("\.")[0];
+      var cite = document.domain;
+      if (cite.indexOf('www.') == 0) {
+          cite = cite.slice(4);
+      }
```

Comment:
Ok. I can't really be sure about that until i deploy. I am going to merge and stage this to see. If it is pulling the wrong domain I'll open a new ticket.

Also, I am going to change `==` to `===` which I should have here